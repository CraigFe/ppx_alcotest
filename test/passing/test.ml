[%%test
module Foo = struct
  let%test f () = ()
  let%test g () = ()
end]
