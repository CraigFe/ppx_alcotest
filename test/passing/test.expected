let _ppx_alcotest_suite = []
include
  struct
    module Foo =
      struct
        let f () = ()
        let g () = ()
        let _ppx_alcotest_suite =
          [Alcotest.test_case "f" `Quick f; Alcotest.test_case "g" `Quick g]
      end
    let _ppx_alcotest_suite = ("Foo", Foo._ppx_alcotest_suite) ::
      _ppx_alcotest_suite
  end
let () = Alcotest.run __FILE__ (Stdlib.List.rev _ppx_alcotest_suite)
